mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientidentifiers/ClientIdentifiersFragment.java

```diff
@@ -203,11 +214,14 @@ public boolean onMenuItemClick(MenuItem item) {
     }
 
     @Override
-    public void identifierDeletedSuccessfully(int position) {
+    public void identifierDeletedSuccessfully(final int position) {
+
+
```

Comment:
@neeraj0403 Can you remove these extra lines which are not necessary.

Reply:
@RajaVamsi11 do i have to remove those  lines which are extra spaces with no content?

Reply:
Yes because they are not necessary.